apps/dashboard/src/components/primitives/form/avatar-picker.tsx

```diff
@@ -25,25 +31,31 @@ const predefinedAvatars = [
   `${window.location.origin}/images/error-warning.svg`,
 ];
 
-type AvatarPickerProps = React.InputHTMLAttributes<HTMLInputElement>;
+type AvatarPickerProps = {
+  name: string;
+  value: string;
+  onChange?: (value: string) => void;
+};
 
-export const AvatarPicker = forwardRef<HTMLInputElement, AvatarPickerProps>(({ id, ...props }, ref) => {
+export const AvatarPicker = forwardRef<HTMLInputElement, AvatarPickerProps>(({ name, value, onChange }, ref) => {
+  const { step } = useStepEditorContext();
+  const variables = useMemo(() => (step ? parseStepVariablesToLiquidVariables(step.variables) : []), [step]);
```

Comment:
the component was missing `variables` and the CodeMirror editor field for the avatar URL